packages/react-components/lib/dispensers/dispenser-accordion.tsx

```diff
@@ -65,6 +76,18 @@ export const DispenserAccordion = React.forwardRef(
     const { dispenserState, dispenser, ...otherProps } = props;
     debug(`render ${dispenser}`);
     const classes = useStyles();
+    const overrideClasses = overrideStyles();
+
+    function usePrevDispenserState(dispenserState: RomiCore.DispenserState | null) {
+      const ref = useRef<RomiCore.DispenserState | null>(null);
+      useEffect(() => {
+        if (dispenserState) {
+          ref.current = dispenserState;
+        }
+      });
+      return ref.current;
+    }
+    const previousState = usePrevDispenserState(dispenserState);
```

Comment:
Not for now and just a comment, if we are going to use the error state and the same logic for other dashboard components as well, maybe creating a custom hook to handle this should be a good idea.

Reply:
Good Idea!

Depending on how the states are going to be handled in future, we may or may not need this, so I will make a comment and keep it as it is for now